### ISSUE TITLE
Still waiting...

### ISSUE THREAD

**Issue by iCodexDev** [RamiLego4Game/luajit-tdlib#2]
Hi, I'm still waiting for using your library <3.
Please coding a library that has less server usage than similar libraries. Thank you.

**Comment by Rami-Sabbagh** [comment]
Hey, do you still want this library?
The issue is I've switched into Java, but if you really want this, I can pull time to finish it.

**Comment by iCodexDev** [comment]
Yes I do, I and all my friends need a good library in the language of Lua if you can complete this library, so that the lowest consumption of server resources in high capacity updates Receive a telegram. In addition, if possible, make all the callbacks in such a way that there is no need to function and receive it. for example: local res = getUser (id) if res.id then ..... end Like the example above, no longer use the function to receive callback.

**Comment by iCodexDev** [comment]
Hello, how is update and continuation of the work going ?? Has the project been canceled? We are still waiting ...
